daemon/src/rollover_maker.rs

```diff
@@ -52,6 +53,7 @@ struct RolloverFailed {
 
 pub struct Actor {
     order_id: OrderId,
+    position: Position,
```

Comment:
This actor should also retrieve the position by returning it from `accept_rollover_proposal`.

Reply:
You're right!!!!